app/controllers/api/account.php

```diff
@@ -1237,7 +1237,7 @@
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_TOKEN)
     ->label('abuse-limit', 10)
-    ->label('abuse-key', 'url:{url},phone:{param-phone}')
+    ->label('abuse-key', ['url:{url},phone:{param-phone}', 'ip:{ip}'])
```

Comment:
```suggestion
    ->label('abuse-key', ['url:{url},phone:{param-phone}', 'ip:{ip}'])
```

don't we need the URL in the IP based abuse key as well ?
Else requests to other endpoints using the same IP may also be blocked 🤔